sdk/identity/Azure.Identity/src/AzureArcManagedIdentitySource.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.Identity
+{
+    internal class AzureArcManagedIdentitySource : ManagedIdentitySource
+    {
+
+        private const string IdentityEndpointInvalidUriError = "The environment variable IDENTITY_ENDPOINT contains an invalid Uri.";
+        private const string NoChallengeErrorMessage = "Did not receive expected WWW-Authenticate header in the response from Azure Arc Managed Identity Endpoint.";
+        private const string InvalidChallangeErrorMessage = "The WWW-Authenticate header in the response from Azure Arc Managed Identity Endpoint did not match the expected format.";
+        private const string ArchApiVersion = "2019-11-01";
```

Comment:
```suggestion
        private const string ArcApiVersion = "2019-11-01";
```

Reply:
Also, we just got confirmation from the service team that we can target the latest version of `2020-06-01`